script/tool/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.3.0+1
```

Comment:
Why is another version bump necessary?

Reply:
you are right, it is unnecessary. I'll revert it